scripts/dist.sh

```
@@ -12,10 +12,10 @@
~
~cd dist/build/2.*.*/
~rm -rf koffi_*bsd*
~rm -rf *ia32
~rm -rf *koffi_win32*
~rm -rf koffi_linux_arm32hf
 rm -rf koffi_linux_riscv64hf64
 
 cd ../../out
-ln -s ../../installer.sh
+ln -s ../../installer.sh # FIXME
```

Shellcheck (0.10.0)

[warning] This ln has no destination. Check the arguments, or specify '.' explicitly.

(SC2226)